sdk/core/Azure.Core/tests/HttpClientTransportTests.cs

```diff
@@ -168,6 +169,11 @@ protected override Task SerializeToStreamAsync(Stream stream, TransportContext c
                 return Task.CompletedTask;
             }
 
+#if NET5_0
+            protected override void SerializeToStream(Stream stream, TransportContext context, CancellationToken cancellationToken)
```

Comment:
No reason to implement?

Reply:
Correct, this implementation is used purely to check whether the content is disposed.